ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryCleanupService.java

```diff
@@ -94,6 +94,7 @@ public void addCleanupTask(final QueryCleanupTask task) {
   public static class QueryCleanupTask implements Runnable {
     private final String appId;
     private final String queryTopicPrefix;
+    private final String altQueryTopicPrefix;
```

Comment:
nit: can you leave a quick comment here explaining why we need to check and possibly delete this "alt prefix"?